### DISCUSSION TITLE
Are Extensions available in Pydeck?

### DISCUSSION THREAD

**Discussion by BlackOilCoder** [visgl/deck.gl#8329] [Q&A]
I'm able to get my terrain layer and geoJSON layers working using pydeck, but I really need to be able to use the TerrainExtension to get the polygons to map to the terrain layer.  Are these exposed in pydeck?  I'm thinking no as I can't find anything about it in the documentation, but I figured I'd ask.  I'm trying to force this app to work in Streamlit, so I'm bound to pydeck.

**Comment by glostis** [comment]
Same here, I'm trying to get the `CollisionFilterExtension` to work with a `TextLayer` in `pydeck`, with no success so far.

**Comment by edwardghbaker** [comment]
Im looking to combine terrain layers and geojson layers in pydeck, I beleive this requires extensions. Does anyone know when these will be available?
